### ISSUE TITLE
goblin engneering quest broken 

### ISSUE THREAD

**Issue by mayras321** [dalaranwow/dalaran-wow#5107]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. I did the quest to get goblin engineering  an every time I turn in the quest the quest just comes right back  so I did this 5 times. so tried doing the gnomish quest an apon compleating quest both quest are gone now an I am neather gnomish or goblin  

2. I should turn in the goblin quest an became a goblin engineering but in the spell book it don't say im eather im just engineering 
3. (Steps to reproduce the problem) I don't know what this means I do not understand 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mayras321** [comment]
Is this at te right place to get the problem fix   Because I’m geting no feedback from any one

**Comment by Lothloryen** [comment]
#5627
